### PR TITLE
Add scyjava recipe

### PR DESCRIPTION
**Pull request by hadim**
Need #7065

### CODE REVIEW

**Comment by hadim** [comment]
- pyjnius: https://github.com/conda-forge/pyjnius-feedstock/pull/1
- jgo: https://github.com/conda-forge/staged-recipes/pull/7065
- scyjava: https://github.com/conda-forge/staged-recipes/pull/7066
- imagej: https://github.com/conda-forge/staged-recipes/pull/7067
- imglyb: https://github.com/conda-forge/staged-recipes/pull/7068

File: recipes/scyjava/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6eb1218e763c4842a114adc5f5d945f7e728a06016e83b9c0e287784bda9eedc
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vvv"
```
Comment (hanslovsky) [inline recipes/scyjava/meta.yaml]: Should we install with `--no-deps` flag?
```suggestion
  script: "{{ PYTHON }} -m pip install . -vvv  --no-deps"
```

**Comment by hanslovsky** [comment]
Confirming that I am OK with being a maintainer for this recipe.

**Comment by hadim** [comment]
@ctrueden are you ok being a maintainer?

**Comment by ctrueden** [comment]
Yes, I would like to be a maintainer of scyjava.

**Comment by hadim** [comment]
@scopatz could you review please?

**Pull request merged**